### PR TITLE
[strings] Update the long tap app tip and sync prison translations

### PR DESCRIPTION
**Pull request by pastk**
- https://github.com/organicmaps/organicmaps/issues/9463#issuecomment-2422175390
- https://github.com/organicmaps/organicmaps/pull/9475#discussion_r1802282818

### CODE REVIEW

File: android/app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -794,7 +794,7 @@
     <!-- App tip #03 -->
     <string name="app_tip_03">Si conoce a algún desarrollador o desarrolladora de software, le puede pedir que implemente alguna funcionalidad que usted necesite.</string>
     <!-- App tip #04 -->
-    <string name="app_tip_04">¿Sabía que puede pulsar prolongadamente sobre cualquier lugar en el mapa para selecionarlo?</string>
+    <string name="app_tip_04">Toque en cualquier lugar del mapa para seleccionar cualquier cosa. Mantenga pulsado para ocultar y volver a mostrar la interfaz.</string>
```
Comment (andergd) [inline android/app/src/main/res/values-es/strings.xml:797]: I would change "cosa" for "elemento", more formal.

File: android/app/src/main/res/values-eu/strings.xml [hunk 2]
```diff
@@ -789,7 +789,7 @@
     <!-- App tip #03 -->
     <string name="app_tip_03">Software garatzaile bat ezagutzen baduzu, behar duzun funtzio bat ezartzeko eska diezaiokezu.</string>
     <!-- App tip #04 -->
-    <string name="app_tip_04">Ba al dakizu mapako edozein toki luze sakatu dezakezula hura hautatzeko?</string>
+    <string name="app_tip_04">Sakatu mapako edozein tokitan edozer hautatzeko. Luze sakatu interfazea ezkutatzeko eta berriro erakusteko.</string>
```
Comment (andergd) [inline android/app/src/main/res/values-eu/strings.xml:792]: Ok

File: android/app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (pastk) [reply android/app/src/main/res/values-es/strings.xml]: thanks!

**Review by ProgramminCat — APPROVED** [review]

**Review by vng — APPROVED** [review]
These tips are used in ```R.array.app_tips``` now.

**Pull request merged**